phpBB/install/phpbbcli.php

```diff
@@ -39,6 +39,9 @@
 // Enable superglobals for cli support
 $phpbb_installer_container->get('request')->enable_super_globals();
 
+// Load compatibility globals to access deprecated globals
+require($phpbb_root_path . 'includes/compatibility_globals.' . $phpEx);
```

Comment:
The main reason I had not to was it requires the global `$phpbb_container`. Which does not exist here. The installer here has it's own version of the container `phpbb_installer_container` . I don't know if it's any different. I also don't see the point of calling that function and having it define all those globals it does, when it's not used/needed here. Allz we need are the 3 constants.